dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -242,6 +247,14 @@ public class ConnectorArguments extends DefaultArguments {
           .ofType(Integer.class)
           .describedAs("N");
 
+  private final OptionSpec<ChronoUnit> optionQueryLogInterval =
+      parser
+          .accepts(OPT_QUERY_LOG_INTERVAL, "The interval between each query log dump")
```

Comment:
"query log dump" is a bit vague to me. I would suggest: "The interval for rotating query log files".